src/XMakeCommandLine/XMake.cs

```diff
@@ -2983,14 +2983,8 @@ private static void ShowHelpMessage()
             Console.WriteLine(AssemblyResources.GetString("HelpMessage_19_IgnoreProjectExtensionsSwitch"));
             Console.WriteLine(AssemblyResources.GetString("HelpMessage_24_NodeReuse"));
             Console.WriteLine(AssemblyResources.GetString("HelpMessage_25_PreprocessSwitch"));
-
             Console.WriteLine(AssemblyResources.GetString("HelpMessage_26_DetailedSummarySwitch"));
-
```

Comment:
For consistency with the rest of the disabled feature, bracket this with

```c#
#if FEATURE_MSBUILD_DEBUGGER
...
#endif
```

instead of removing it entirely. We're still trying to gauge the reaction from people about removing the debugger.